### PR TITLE
Automated migration to v0.7.0 using scalafix

### PR DESCRIPTION
**Pull request by jto**
Add missing imports in files and rewrite `sys.prop` calls.

### CODE REVIEW

File: scalafix/project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=1.1.1
```
Comment (regadas) [inline scalafix/project/build.properties]: 🎨 is it ok if we use same version as in scio?

Comment (jto) [reply scalafix/project/build.properties]: Probably yes. I used the g8 template scalafix provide but I should be able to change that.

File: scalafix/build.sbt [hunk 2]
```diff
@@ -0,0 +1,54 @@
+lazy val V = _root_.scalafix.sbt.BuildInfo
```
Comment (regadas) [inline scalafix/build.sbt]: can we make this part of scio build instead of different project?

**Comment by jto** [comment]
FYI: I tested it on a pretty big pipeline at Spotify and it gets the compilations errors down from 329 to 6. Fixing those errors takes less than 5 min.

File: scalafix/build.sbt (hunk 2 above)
Comment (jto) [reply scalafix/build.sbt]: Probably not. The build for scalafix rules is pretty specific. I followed their tutorial and creating a separate project seems to be the recommended method

**Comment by jto** [comment]
I need to add a migration rule for methods that use a parameter and a potentially explicit`Ordering` like `SCollection.quantilesApprox` because

```scala
scoll.quantilesApprox(numQuantiles)(Ordering.by(_.smth))
```

is now

```scala
scoll.quantilesApprox(numQuantiles, Ordering.by(_.smth))
```

**Comment by jto** [comment]
@regadas I think we can merge this and improve it later if necessary

**Comment by regadas** [comment]
@jto Yup let's merge this, We will need to pimp a little bit the build info, just in case we decide to publish these rules.

**Pull request merged**